agent/agent.go

```diff
@@ -312,6 +312,7 @@ func toEnv(w *model.Work) map[string]string {
 	}
 	if w.Build.Event == model.EventDeploy {
```

Comment:
this value can be set for all event types, so it can be moved outside this `if` block